src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -68,6 +72,29 @@ public string DisplayName
         }
     }
 
+    /// <summary>
+    ///   Description of processes an organelle does if any.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     This supports custom format (for example: "Turns [glucose] into [atp]") where strings inside
+    ///     the square brackets will be parsed and replaced with a predefined template. This is done to
+    ///     make translating feasible.
+    ///     NOTE: description string should only be set here and not directly on the rich text label node
+    ///     as it will be overidden otherwise.
+    ///   </para>
+    /// </remarks>
+    [Export]
+    public string ProcessesDescription
+    {
+        get => processesDescription;
+        set
+        {
+            processesDescription = value;
```

Comment:
I think checking if the new value is the same as old would be worth it here to avoid expensive operation if it the text didn't change.